spec/integration/cleanup_mode_spec.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+require 'codeowners/checker'
+
+RSpec.describe 'Report mode' do
+  def codeowners_file_body
+    File.open(File.join(IntegrationTestRunner::PROJECT_PATH, '.github', 'CODEOWNERS')).read
+  end
+
+  def owners_file_body
+    File.open(File.join(IntegrationTestRunner::PROJECT_PATH, '.github', 'OWNERS')).read
+  end
+
+  subject(:runner) do
+    IntegrationTestRunner
+      .new(codeowners: codeowners, owners: owners, file_tree: file_tree)
+      .run(command: 'cleanup', flags: [])
+  end
+
+  let(:codeowners) { [] }
+  let(:owners) { [] }
+  let(:file_tree) { {} }
+
+  context 'when no issues' do
+    let(:codeowners) do
+      ['',
+       'lib/new_file.rb @mpospelov']
```

Comment:
@mpospelov is just getting credits for it 😄 

Let's go with `@user` or `@company/team` ;)

Reply:
I've been thinking and maybe this whole spec doesn't even make sense anymore.
The original spec was checking that no errors were reported, but I decided that was weird to report errors and rewrite the entire file because then it would raise the question of whether the check was done before or after the rewrite.

So this spec is not testing anything, maybe we should just remove it. Thoughts?

Reply:
Sure! just do it :)